### ISSUE TITLE
Error while installing

### ISSUE THREAD

**Issue by jcubic** [raviqqe/schemat#59]
I've tried to install the tool on Fedora, but got this error:

```
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /home/kuba/.cargo/registry/src/index.crates.io-6f17d22bba15001f/mfmt-0.2.3/src/lib.rs:2:12
  |
2 | #![feature(allocator_api)]
  |            ^^^^^^^^^^^^^

error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /home/kuba/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bumpalo-3.14.0/src/lib.rs:5:48
  |
5 | #![cfg_attr(feature = "allocator_api", feature(allocator_api))]
  |                                                ^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0554`.
error: could not compile `mfmt` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
error: could not compile `bumpalo` (lib) due to previous error
error: failed to compile `schemat v0.1.21`, intermediate artifacts can be found at `/tmp/cargo-installRjjJLo`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

**Comment by raviqqe** [comment]
Ah, you actually need nightly Rust. I'm gonna add it to readme.

**Comment by raviqqe** [comment]
Closed by #61.
